firmware/application/apps/ui_level.cpp

```diff
@@ -62,18 +81,19 @@ LevelView::LevelView(NavigationView& nav)
                   &freq_stats_rssi,
                   &freq_stats_db,
                   &freq_stats_rx,
-                  &audio_mode,
+                  &field_audio_mode,
                   &peak_mode,
                   &rssi,
                   &rssi_graph});
 
+    radio_mode = NFM_MODULATION;
+    audio_mode = 0;
+
     // activate vertical bar mode
     rssi.set_vertical_rssi(true);
-    // activate counters for RxSat
-    shared_memory.request_m4_performance_counter = 2;
 
-    change_mode(NFM_MODULATION);              // Start on AM
-    field_mode.set_by_value(NFM_MODULATION);  // Reflect the mode into the manual selector
+    change_mode(radio_mode);              // Start on AM
```

Comment:
Comment doesn't seem to match mode on line 89?

Reply:
Changed so it's saved and restored